src/main/kotlin/fi/metatavu/timebank/api/controllers/PersonsController.kt

```diff
@@ -47,10 +47,14 @@ class PersonsController {
             throw Error("Invalid minimumBillableRate!")
         }
 
+        if (person.unspentVacations < 0 || person.spentVacations < 0){
+            throw Error("Invalid vacation days!")
+        }
+
```

Comment:
This will cause weird behaviours if these are for some reason set to the default of this PR, -1.